pkg/devfile/adapters/docker/component/adapter.go

```diff
@@ -219,6 +220,9 @@ func (a Adapter) Delete(labels map[string]string) error {
 		}
 	}
 
+	spinner := log.Spinner(fmt.Sprintf("Deleting devfile component %s", componentName))
+	defer spinner.End(false)
+
```

Comment:
This seems to be repeated again for the kubernetes adapter. Can we move it before the call?

Reply:
See https://github.com/openshift/odo/pull/3405#discussion_r447820362